Blish HUD/Controls/Container.cs

```diff
@@ -146,6 +196,7 @@ public abstract class Container : Control, IEnumerable<Control> {
 
         public bool AddChild(Control child) {
             if (_children.Contains(child)) return true;
+            if (_panelScrollbar != null && child == _panelScrollbar) return true;
```

Comment:
Can be shortened to

`return _panelScrollbar != null && child == _panelScrollbar`

Reply:
Wouldn't that return always and not continue to add the child?